spec/models/content_asset_spec.rb

```diff
@@ -1,5 +1,30 @@
 require "rails_helper"
 
 RSpec.describe ContentAsset, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "with an attached file" do
+    before(:each) do
+      @content_asset = ContentAsset.new(title: "Title")
+      @content_asset.avatar.attach(io: File.open("spec/fixtures/sample.jpeg"), filename: "sample.jpeg", content_type: "image/jpeg")
+      @content_asset.save!
+    end
+
+    it "has a file attached" do
+      expect(@content_asset.avatar).to be_attached
+    end
+  end
+
+  describe "required attributes" do
+    it "requires a title" do
```

Comment:
Same as above regarding use of explicit subject

Reply:
Yep explicit